unittests/Feature/FeatureModel.cpp

```diff
@@ -45,20 +34,19 @@ TEST(FeatureModel, cloneUnique) {
 
 TEST(FeatureModel, cloneRoot) {
   FeatureModelBuilder B;
-  B.makeFeature<BinaryFeature>("a");
-  B.setRoot("a");
+  B.makeFeature<RootFeature>("a");
+  B.setRootName("a");
   auto FM = B.buildFeatureModel();
   assert(FM);
 
   auto Clone = FM->clone();
   assert(Clone);
   FM.reset();
 
-  // The inner EXPECT_TRUE just handles the nodiscard of isRoot
+  // The inner EXPECT_TRUE just handles the nodiscard of getParent
   // NOLINTNEXTLINE
-  ASSERT_EXIT(EXPECT_TRUE(FM->getFeature("a")->isRoot()),
-              testing::KilledBySignal(SIGSEGV), ".*");
-  EXPECT_TRUE(Clone->getFeature("a")->isRoot());
+  EXPECT_DEATH(EXPECT_TRUE(FM->getFeature("a")->getParent()), ".*");
+  EXPECT_FALSE(Clone->getFeature("a")->getParent());
```

Comment:
Shouldn't this become a `isa` check for RootNode instead of an empty parent check?

Reply:
Indeed.